src/ripple/protocol/TxFormats.h

```diff
@@ -56,6 +56,8 @@ enum TxType {
     ttTRUST_SET = 20,
     ttACCOUNT_DELETE = 21,
 
+    ttHOOK_SET = 22,
```

Comment:
Consider:

    ttHOOK_SET = 22 [[maybe_unused]],

Since the code would be currently unused.

Reply:
I think the correct syntax for setting an attribute on enum member is
```c++
ttHOOK_SET [[maybe_unused]] = 22,
```
there's a few other unused tx types, should we attribute all of them?

Reply:
Which other tx types are unused?

Reply:
6, 9, 10, 11, 102 I guess?

Reply:
I decided to let you guys work out which others to attribute (if any) in another PR